WordPress/build.gradle

```diff
@@ -249,7 +246,7 @@ dependencies {
     })
 
     implementation 'com.android.volley:volley:1.1.1'
-    implementation 'com.google.firebase:firebase-messaging:21.1.0'
+    implementation 'com.google.firebase:firebase-messaging:20.1.5'
```

Comment:
Is this version _regression_ expected?
Since the CI failures appear to be on code related to GCM, I'm guessing that might be related 🤔

Reply:
This change was definitely not intentional and I didn't see this in the Github UI when I checked this out earlier 🤔 In any case I'll revert this change 👍